backend/fms_core/resources/sample.py

```diff
@@ -198,10 +216,19 @@ def import_field(self, field, obj, data, is_m2m=False):
             obj.collection_site = get_normalized_str(data, "Collection Site")
 
         elif field.attribute == "sample_kind_name":
-            obj.sample_kind = SampleKind.objects.get(name=data["Sample Kind"])
+            try:
+                obj.sample_kind = SampleKind.objects.get(name=data["Sample Kind"])
+            except Exception as e:
+                pass
```

Comment:
Still unsure about passing the exception here too..

Reply:
Definitely! my bad. Forgot to raise a ValueError. 

`raise ValueError("Sample kind name cannot be blank.")`